test/unit/overall-result.test.js

```diff
@@ -119,23 +98,11 @@ describe('calculate overall results', () => {
     const result = calculate(155000)
     expect(result.overallResult[0].payment).toBeGreaterThan(result.overallResult[1].payment)
     expect(result.overallResult[1].payment).toBeGreaterThan(result.overallResult[2].payment)
-    expect(result.overallResult[2].payment).toBeGreaterThan(result.overallResult[3].payment)
   })
 
   test('returns overall reductions get higher', () => {
     const result = calculate(1000)
-    expect(result.overallResult[3].payment).toBeLessThan(result.overallResult[2].payment)
     expect(result.overallResult[2].payment).toBeLessThan(result.overallResult[1].payment)
     expect(result.overallResult[1].payment).toBeLessThan(result.overallResult[0].payment)
   })
```

Comment:
Would be good to circle back on the tests at a later date to see if these can be streamlined to make visualising and editing them for changes any more intuitive.

Reply:
Created the following ticket to review status quo:
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/238481